src/components/LibrariesList.tsx

```diff
@@ -1,67 +1,27 @@
 import * as React from "react";
-import { Store, Reducer } from "redux";
+import { Store } from "redux";
 import { State } from "../reducers/index";
-import { LibrariesData, LibraryData } from "../interfaces";
-import { connect } from "react-redux";
-import ActionCreator from "../actions";
+import { LibraryData } from "../interfaces";
 import LibrariesListItem from "./LibrariesListItem";
 
-export interface LibrariesListStateProps {
-  libraries?: LibrariesData;
-}
-
 export interface LibrariesListOwnProps {
+  libraries?: LibraryData[];
   store?: Store<State>;
 }
 
-export interface LibrariesListDispatchProps {
-  fetchData: () => Promise<LibrariesData>;
-}
-
-export interface LibrariesListProps extends LibrariesListStateProps, LibrariesListOwnProps, LibrariesListDispatchProps {};
-
-export class LibrariesList extends React.Component<LibrariesListProps, {}> {
-
+export default class LibrariesList extends React.Component<LibrariesListOwnProps, {}> {
   render(): JSX.Element {
-    let hasLibraries = (this.props.libraries && this.props.libraries.libraries && this.props.libraries.libraries.length > 0);
-    return(
-      <ul className="list">
-        { hasLibraries ?
-          this.props.libraries.libraries.map(library =>
-            <LibrariesListItem
-              key={library.uuid}
-              library={library}
-              store={this.props.store}
-            />
-          ) :
-          <span className="page-header">There are no libraries in this registry yet.</span>
-        }
-      </ul>
+    return (
+          this.props.libraries ?
```

Comment:
The indentation for this is very far.

Reply:
I tried indenting it a few different ways, and they all look wrong to me; do you have a preference?